orchestrator-salt/src/main/resources/salt-common/salt/monitoring/scripts/logging-agent.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if [[ -f "/var/run/cdp-logging-agent/cdp-logging-agent.pid" ]]; then
+  platform="{{ salt['pillar.get']('fluent:platform') }}"
+  logging_types=""
+{%- if salt['pillar.get']('fluent:cloudStorageLoggingEnabled') %}
+  if [[ "$platform" == "AWS" ]]; then
+    logging_types="s3"
+  elif [[ "$platform" == "AZURE" ]]; then
+    logging_types="abfs"
+  elif [[ "$platform" == "GCP" ]]; then
+    logging_types="gcs"
+  fi
```

Comment:
can't we get this from pillar and construct the script accordingly? this would always result in the same